### ISSUE TITLE
[Ubiquiti][Wifi 5Ghz] Possible fix for slow speeds in AC wifi.

### ISSUE THREAD

**Issue by maxi1134** [ValveSoftware/SteamOS#843]
#### Your system information

* Steam client version: same as below
* SteamOS version:  will edit back when at me deck
* Opted into Steam client beta?: Yes
* Opted into SteamOS beta?: Yes
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:

Hi guys,
After spending the last 8 hours on this, I found out that the steamdecks speeds are 30x faster when using the lower channels of my 5ghz Antennas. (This is replicated across 4 APs).

Basically, using any channel Post-DFS will cause me to only reach 15-20mbps speeds.
Reducing the channel to pre-DFS channels fixed this issue altogether.

If anyone could corroborate, that would be great.

I also think that Steam should take a look at this, as many people are set on higher channels than DFS to avoid interference,

**Comment by Gwindalmir** [comment]
I have a Netgear R7000, running tomato firmware.
I normally use channel 149, lower control channel, and I get the max speed of my WAN connection, ~100 Mbps, or 11.4 MB/s.
I tried to change it to channel 36, and my speed dropped to 6MB/s.

I don't use auto-channel selection, never have.
I don't know if this information is useful or not. I'm not a radio guru, so while I know some about 2.4GHz, I don't know much about 5GHz stuff.

**Comment by Tri125** [comment]
I can confirm that I have the same issue as you @maxi1134 on the Steam Machine in 2026. I haven't taken the time to confirm if the issue is still present on the Steam Deck.

**System information**

- Steam client version: 1787790984
- SteamOS version: 3.8.26
- Opted into Steam client beta?: I'm not using the experimental SteamRT3 client if that's what this question is about.
- Opted into SteamOS beta?: Yes
- Have you checked for updates in Settings > System?: Yes

My router is a UniFi Dream Machine, specifically the model from 2019 that only support 2.4 GHz and 5 GHz.
I believe the problem also exist on the SteamDeck because I experienced similar speed the last time I used the device, which was less than a year ago.

I confirmed that my Steam Machine was connected on the 5 GHz band before proceeding.

On Channel 60 (5 GHz, 80 MHz) I was capped between 20 Mbps and 27 Mbps.

When I forced my router to use Channel 48 (5 GHz, 80 MHz) and let the Steam Machine reconnect to the 5GHz band, the download speed went up to 120 Mbps.

There's definitively a problem when a DFS channel is used.
